pages/linux/vso.md

```diff
@@ -0,0 +1,36 @@
+# vso
+
+> Utility to perform maintenance tasks on Vanilla OS.
+> More information: <https://github.com/Vanilla-OS/vanilla-system-operator>.
+
+- Trigger a system update immediately:
+
+`sudo vso trigger-update --now`
+
+- Create an automated task upon an application's launch:
+
+`vso create-task --name "{{string}}" --description "{{string}}" --{{command|on-condition-command}} "{{command}}" --on-process {{integer}}`
+
+- Create an automated task to execute during boot:
+
+`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --on-boot`
+
+- Create an automated task to execute during specific battery states:
+
+`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --{{on-low-battery|on-charge|on-battery|on-full-battery}}`
+
+- Create an automated task to execute at a specific time interval (hh: mm) or re-execute it every time after specific time intervals (m,h,d):
+
+`vso create-task --name "{{string}}" --description "{{string}}" --command "{{command}}" --{{at|every}} {{time_interval}}`
```

Comment:
I think this example should be split into 2. is the time in 24hs format or am/pm?

Reply:
Yeah, but the page already has 8 examples, maybe I will remove this one and change it to a different one for update-check.

It's in 24 hours format afaik.

Reply:
Done, updated it.